lib/content-services/src/lib/content-metadata/services/content-metadata.service.ts

```diff
@@ -86,14 +85,26 @@ export class ContentMetadataService {
         return groupedProperties;
     }
 
-    setTitleToNameIfNotSet(propertyGroups: OrganisedPropertyGroup[]): OrganisedPropertyGroup[] {
+    private setTitleToNameIfNotSet(propertyGroups: OrganisedPropertyGroup[]): OrganisedPropertyGroup[] {
         propertyGroups.map((propertyGroup) => {
             propertyGroup.title = propertyGroup.title || propertyGroup.name;
         });
         return propertyGroups;
     }
 
-    filterEmptyPreset(propertyGroups: OrganisedPropertyGroup[]): OrganisedPropertyGroup[]  {
+    private filterEmptyPreset(propertyGroups: OrganisedPropertyGroup[]): OrganisedPropertyGroup[]  {
         return propertyGroups.filter((props) => props.properties.length);
     }
 }
+
+@Injectable({
+    providedIn: 'root',
+    useClass: ContentMetadataServiceImpl
+})
+export abstract class ContentMetadataService {
```

Comment:
Basic idea is to just extract a service interface and keep the current implementation as default.
Kept it in the same file to avoid breaking any existing import but ideally these would live in separate files at least.